lib/core-common/src/types.ts

```diff
@@ -262,13 +262,7 @@ export type Preset =
  */
 export type Entry = string;
 
-type StorybookRefs = Record<
-  string,
-  {
-    title: string;
-    url: string;
-  }
->;
+type StorybookRefs = Record<string, { title: string; url: string } | { disable: boolean }>;
```

Comment:
```suggestion
type StorybookRefs = Record<string, { title: string; url: string; disable?: boolean }>;
```

Reply:
That was my initial solution but it means you have to set a title and URL, which is weird if you're trying to disable it.

Reply:
ok discussed offline & this is for autorefs